### PR TITLE
chore: fix qrc resource bloat

### PR DESCRIPTION
**Pull request by caybro**
let the `generate-rcc.go` skip files under `vendor` and `tests` subdirs; decreases the binary size and thus RAM usage by ~130 MB!

It became more pronounced recently with the import of `ui/StatusQ/vendor/qzxing` git submodule
```
$ ls -al resources.*
-rw-r--r--. 1 ltinkl ltinkl  81950109 Mar  8 14:47 resources.rcc
-rw-r--r--. 1 ltinkl ltinkl 215385428 Mar  8 12:35 resources.rcc.bloat
```

### What does the PR do

Reduces binary resources/app size

### Screenshot of functionality (including design for comparison)

- [N/A] I've checked the design and this PR matches it

### CODE REVIEW

**Review by alexjba — APPROVED** [review]
🎉

File: ui/generate-rcc.go [hunk 1]
```diff
@@ -52,6 +52,9 @@ func main() {
 			if err != nil {
 				return err
 			}
+            if info.IsDir() && (info.Name() == "vendor" || info.Name() == "tests") {
+                return filepath.SkipDir
+            }
```
Comment (0x-r4bbit) [inline ui/generate-rcc.go:57]: This looks like this needs to run through a formatter

**Review by 0x-r4bbit — APPROVED** [review]

File: ui/generate-rcc.go (hunk 1 above)
Comment (caybro) [reply ui/generate-rcc.go:57]: How to do this? My first ever piece of `go` code, bear with me :)

Comment (0x-r4bbit) [reply ui/generate-rcc.go:57]: There's a formatting tool something like

```sh
gofmt -w yourcode.go
```

https://go.dev/blog/gofmt

Congrats on your first go code!

Comment (caybro) [reply ui/generate-rcc.go]: Done

**Review by richard-ramos — APPROVED** [review]

**Pull request merged**